service_stub.arrai

```diff
@@ -0,0 +1,3 @@
+let [_, basepath, json, appname, groups] = //os.args;
```

Comment:
Nit: Can you add a comment explaining what this file is for? A reader could be forgiven for thinking it's a pointless alias.

Ideally, if it's only temporary until we build some more arr.ai features, link to the issue that would obsolete it.

Reply:
This file serves as a remote stub for service.arrai which is hosted in another repository. I don't want to maintain a copy of service.arrai here and let it drift away from the remote copy.

Reply:
Added the above as comments.

Reply:
Can you also reference https://github.com/arr-ai/arrai/issues/504 and mention this stub will not be necessary once arr.ai can run remote scripts? It makes arr.ai look pretty dumb otherwise.

Reply:
ok